src/Core/Auth/Identity/DuoUniversalPromptService.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Bit.Core.Auth.Models;
+using Bit.Core.Context;
+using Bit.Core.Entities;
+using Bit.Core.Settings;
+using Duo = DuoUniversal;
+
+namespace Bit.Core.Auth.Identity;
+
+/* 
+    PM-5156 addresses tech debt
+    Interface to allow for DI, will end up being removed as part of the removal of the old Duo SDK v2 flows.
+    This service is to support SDK v4 flows for Duo. At some time in the future we will need
+    to combine this service with the DuoWebTokenProvider and OrganizationDuoWebTokenProvider to support SDK v4.
+*/
+public interface IDuoUniversalPromptService
```

Comment:
Can we add temporary to the name here? Something like `ITemporaryDuoSDKUpgradeService`? The universal prompt is actually separate from our work here. It's more of a upgrade from an iframe implementation to a frameless implementation.